Settings.StyleCop

```diff
@@ -216,24 +231,34 @@
         </Rule>
         <Rule Name="PropertySummaryDocumentationMustOmitSetAccessorWithRestrictedAccess">
           <RuleSettings>
-            <BooleanProperty Name="Enabled">False</BooleanProperty>
+            <BooleanProperty Name="Enabled">True</BooleanProperty>
           </RuleSettings>
         </Rule>
         <Rule Name="ElementDocumentationMustNotBeCopiedAndPasted">
           <RuleSettings>
-            <BooleanProperty Name="Enabled">False</BooleanProperty>
+            <BooleanProperty Name="Enabled">True</BooleanProperty>
           </RuleSettings>
         </Rule>
         <Rule Name="SingleLineCommentsMustNotUseDocumentationStyleSlashes">
           <RuleSettings>
-            <BooleanProperty Name="Enabled">False</BooleanProperty>
+            <BooleanProperty Name="Enabled">True</BooleanProperty>
           </RuleSettings>
         </Rule>
         <Rule Name="DocumentationTextMustNotBeEmpty">
           <RuleSettings>
             <BooleanProperty Name="Enabled">False</BooleanProperty>
           </RuleSettings>
         </Rule>
+        <Rule Name="DocumentationTextMustBeginWithACapitalLetter">
+          <RuleSettings>
+            <BooleanProperty Name="Enabled">False</BooleanProperty>
```

Comment:
I think this should be true to have a well written documentation text.
```suggestion
            <BooleanProperty Name="Enabled">True</BooleanProperty>
```

Reply:
You can check in my commits what it would cause: 
![image](https://user-images.githubusercontent.com/9156103/73003882-bd860d80-3ddc-11ea-99b2-7e9f899b5f67.png)

I agree with you, but I don't want to fix 25 issues in this PR, so that is why I added it to your refactoring list in the description.

Reply:
Got it